src/schema.ts

```diff
@@ -5,8 +5,13 @@ export const typeDefs = `#graphql
   type Query{
     hello: String
     user(input: UserInput!): User
+    users(input: UsersInput): [User!]!
   }
 
+  input UsersInput{
+    maxUsers: Int
```

Comment:
geralmente o termo do número de usuários é `limit`
```suggestion
    limit: Int
```